From 1a764ac793198d30c8ed6b44c9c4859edb28e28e Mon Sep 17 00:00:00 2001
From: Dong Aisheng <b29396@freescale.com>
Date: Thu, 4 Jun 2015 17:20:41 +0800
Subject: [PATCH 0714/1221] MLK-11031 dts: imx6ul-14x14-ddr3-arm2: set
 flexcan2 disabled by default

Set flexcan2 disabled by default due to pin conflict with uart2.
User should user imx6ul-14x14-ddr3-arm2-flexcan2.dtb to enable flexcan2.

Signed-off-by: Dong Aisheng <b29396@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
index e745007..2e634057 100644
--- a/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
+++ b/arch/arm/boot/dts/imx6ul-14x14-ddr3-arm2.dts
@@ -138,7 +138,7 @@
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_flexcan2>;
 	xceiver-supply = <&reg_can2_3v3>;
-	status = "okay";
+	status = "disabled";
 };
 
 &gpc {
-- 
1.7.5.4

